src/routes/Forms.jsx

```diff
@@ -42,26 +40,19 @@ const Forms = () => {
         justifyContent: 'center'
       }}
     >
-      <Button
```

Comment:
Think we could keep the button for clearing selected users here. That way case workers could just unselect them here rather than having to go back to "Clients".

Reply:
fixed